src/webui/src/App.tsx

```diff
@@ -117,39 +123,43 @@ class App extends React.Component<{}, AppState> {
         ];
 
         const reactPropsChildren = React.Children.map(this.props.children, child =>
-            React.cloneElement(
-                child as React.ReactElement<any>, {
+            React.cloneElement(child as React.ReactElement<any>, {
                 interval,
-                columnList, changeColumn: this.changeColumn,
+                columnList,
+                changeColumn: this.changeColumn,
                 experimentUpdateBroadcast,
                 trialsUpdateBroadcast,
-                metricGraphMode, changeMetricGraphMode: this.changeMetricGraphMode,
-                bestTrialEntries, changeEntries: this.changeEntries
+                metricGraphMode,
+                changeMetricGraphMode: this.changeMetricGraphMode,
+                bestTrialEntries,
+                changeEntries: this.changeEntries
             })
         );
 
         return (
-            <Stack className="nni" style={{ minHeight: window.innerHeight }}>
-                <div className="header">
-                    <div className="headerCon">
+            <Stack className='nni' style={{ minHeight: window.innerHeight }}>
+                <div className='header'>
+                    <div className='headerCon'>
                         <NavCon changeInterval={this.changeInterval} refreshFunction={this.lastRefresh} />
                     </div>
                 </div>
-                <Stack className="contentBox">
-                    <Stack className="content">
+                <Stack className='contentBox'>
+                    <Stack className='content'>
                         {/* if api has error field, show error message */}
-                        {
-                            errorList.map((item, key) => {
-                                return (
-                                    item.errorWhere && <div key={key} className="warning">
-                                        <MessageInfo info={item.errorMessage} typeInfo="error" />
+                        {errorList.map((item, key) => {
+                            return (
```

Comment:
Is `return` necessary here?

Reply:
I also think so. We can use arrow function rules to standardize it.